Lib/test/test_exceptions.py

```diff
@@ -1565,21 +1565,87 @@ def test_name_error_bad_suggestions_do_not_trigger_for_small_names(self):
     def test_name_error_suggestions_do_not_trigger_for_too_many_locals(self):
```

Comment:
We must find a way to make this test more bearable, maybe using `compile()` as this is getting a bit wild. :(